apps/starccm/readme.md

```diff
@@ -0,0 +1,65 @@
+## Install and run Fluent Benchmarks
+
+## Prerequisites
+
+Cluster is built with the desired configuration for networking, storage, compute etc. 
+
+Recommended cluster setup
+Start with the simple_hpc_pbs example. Before you do azhpc-build copy the scripts directory from <azurehpc>/apps/starccm to your cluster build directory. First, you will need to add the following line above the "tags" section for the headnode.
+
+"data_disks": [2048, 2048],
+    
+ Second, add the following section in the scripts section above the pbsdownload piece (~line 90 in the config.json file). 
```

Comment:
an alternative (preferred) to requiring to do this during cluster setup would be to run the script on the compute nodes post-setup. see https://github.com/Azure/azurehpc/tree/master/apps/lammps for e.g.

Reply:
Addressed as recommended by the alternative approach